Dockerfile.test

```diff
@@ -23,8 +25,14 @@ WORKDIR /crawler
 
 COPY . /crawler
 
+RUN (cd psvmi/psvmi; python setup.py build && python setup.py install)
+
+COPY psvmi/maps maps
+COPY psvmi/offsets offsets
+COPY psvmi/header.h .
+
 CMD (docker daemon > ../docker.out 2>&1 &) && \
 	sleep 5 && \
         flake8 --max-complexity 10 . || true && \
         pylint crawler || true && \
-        python setup.py test --addopts '--cov=.'
+        python setup.py test --addopts '-s --cov=.'
```

Comment:
'-s' allows verbosity. Original was silent. I only added -s to see output for my test code. Other tests's do print lots of stuff. So off / on (-s) ?